common-libs/vc-common/tsconfig.json

```diff
@@ -20,6 +20,8 @@
     "noFallthroughCasesInSwitch": true,
     "moduleResolution": "node",
     "jsx": "react",
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "outDir": "dist",
+    "skipLibCheck": true
```

Comment:
Why has this been added?

Reply:
Because tsdx added this option internally, and it was needed to compile the projects.

Reply:
Thanks!